docs/.vitepress/config.mts

```diff
@@ -128,7 +133,14 @@ export default withMermaid({
           items: [
             {text: 'Repository Structure', link: '/contributing/structure'},
             {text: 'Clinical Domain Agent', link: '/contributing/clinical-domain-agent'},
-            {text: 'Trust Center Agent', link: '/contributing/trust-center-agent'},
+            {
+              text: 'Trust Center Agent',
+              items: [
+                {text: "Overview", link: '/contributing/trust-center-agent'},
+                {text: 'Pseudonymisierung', link: '/contributing/pseudonymisierung'},
```

Comment:
I'd suggest only one link (with english label) to the german version. In the german version the info text points to the translation, and in the english translation we link back to german, with an info box stating that the page is a translation, and why it is translated.